code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -229,6 +230,75 @@ Contains:
 	species_exception = list(/datum/species/angel)
 	tauric = TRUE		//Citadel Add for tauric hardsuits

```

Comment:
should just clean up my laziness while you're at it. :P

Reply:
<3